mountpoint-s3-client/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "mountpoint-s3-client"
 # See `/mountpoint-s3-client/PUBLISHING_CRATES.md` to read how to publish new versions.
-version = "0.14.0"
+version = "0.14.1"
```

Comment:
It should be a minor bump - not patch - since its adding a new feature. :(

Reply:
For `0.x.y` releases, my understanding is that semver is not too detailed: it only specifies that `x` allows breaking changes. I'm fine with only incrementing `y` here.